src/server/editorServices.ts

```diff
@@ -1616,13 +1616,6 @@ namespace ts.server {
             if (this.seenSurveyProjects.has(project.projectName)) {

```

Comment:
Should we probably remove these methods and maps as well ? (and  the even name for compatibility?)

Reply:
Well, we will need all the same infrastructure for future surveys. The question is how likely we are to need a survey soon.
I can definitely remove the entire thing if you think it's the right thing.